### PR TITLE
Changes the 'multibelt', aka the father of belts and adds mini extinguishers to toolbelts (atmos nerds rejoice)

### PR DESCRIPTION
**Pull request by ghost**
Too high to make a changelog look at the code

I also removed the var edited belt on the map and added two leather belts around the map

### CODE REVIEW

**Comment by ghost** [comment]
There are shitty codersprites included yeah

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+	desc = "A leather belt with extra wide pouches that can hold a variety of items."
 	icon = 'icons/obj/clothing/belts.dmi'
-	icon_state = "utilitybelt"
-	item_state = "utility"
+	icon_state = "leatherbelt"
+	item_state = "leather"
+	storage_slots = 5
 	slot_flags = SLOT_BELT
-	attack_verb = list("whipped", "lashed", "disciplined")
+	attack_verb = list("whipped", "lashed", "smacked")
```
Comment (Atskadan) [inline code/game/objects/items/weapons/storage/belt.dm]: why

File: code/modules/clothing/spacesuits/rig.dm [hunk 2]
```diff
@@ -128,7 +128,7 @@
 //Syndicate Stealth Rig
 /obj/item/clothing/head/helmet/space/rig/syndistealth
 	name = "night-black hardsuit helmet"
-	desc = "An advanced helmet designed for work in covert operations. Property of Gorlex Marauders."
+	desc = "An advanced helmet designed for work in covert operations. Property of the Spider Clan."
```
Comment (Atskadan) [inline code/modules/clothing/spacesuits/rig.dm]: the spider clan isnt affiliated with the syndicate and i doubt they make hardsuits

Comment (ghost) [inline code/modules/clothing/spacesuits/rig.dm]: I felt like adding them to the stealth suit description felt a lot better than the gorlex marauders, aka the blood-red suit mercs. The design fits the spider clan aesthetic a lot better, and there's no saying that the syndicate doesn't get their tech from a ton of sources (evidenced in the weapon descs)

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (ghost) [inline code/game/objects/items/weapons/storage/belt.dm]: it's a verb that should be unique to the chain of command, duh.

File: code/modules/clothing/spacesuits/rig.dm (hunk 2 above)
Comment (Atskadan) [inline code/modules/clothing/spacesuits/rig.dm]: Isn't MI13 the "stealthier" branch of the Syndicate?

Comment (ghost) [inline code/modules/clothing/spacesuits/rig.dm]: I'll look for some refs in the code or the wiki, if so I'll replace it, ja.

**Comment by ghost** [comment]
Updated a TON of hardsuit descriptions to be uniform and better represent what the hardsuits protect against.

Also added DEEP LORE proper syndicate factions to the stealth syndie rigsuit, be sure to rush into the shuttle to be the first one to get at it! (it has slightly better melee but worse bullet and laser protection iirc)

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Atskadan) [inline code/game/objects/items/weapons/storage/belt.dm]: nah cause when you whack someone with a belt you're doing it to discipline them
you know when you see someone take off their belt theyre about to smack the shit out of some kid for being bad

**Comment by Atskadan** [comment]
actually the lore for the syndicate says that MI13 and the GM dont like eachother so maybe it should say it was 'loaned' to them since nuke ops are pretty obviously exclusively GM

**Comment by ghost** [comment]
Honestly I don't think anyone will give that much of a shit plus if an admin ever wants to do an event with MI13 and ebows they can use these, one just being on the shuttle for the lone wolf is just better than that old fashioned fatty space suit in my opinion.

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (ghost) [inline code/game/objects/items/weapons/storage/belt.dm]: that's as dumb as 'home runned'

File: code/game/objects/items/weapons/storage/belt.dm [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
-	name = "belt"
-	desc = "Can hold various things."
+	name = "leather belt"
+	desc = "A leather belt with extra wide pouches that can hold a variety of items."
 	icon = 'icons/obj/clothing/belts.dmi'
-	icon_state = "utilitybelt"
-	item_state = "utility"
+	icon_state = "leatherbelt"
+	item_state = "leather"
+	storage_slots = 5
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/storage/belt.dm:7]: you need to set storage_slots = 7 on the military belt so this doesn't nerf syndies

**Comment by ghost** [comment]
ja, sankyuu

**Comment by ghost** [comment]
gonna change the fire extinguishers to the small version one, don't merge this yet

**Comment by ghost** [comment]
this sexy shit is ready to go unless anyone has ideas

**Pull request merged**